Dockerfile

```
@@ -23,8 +23,7 @@ WORKDIR /go/src/github.com/cosmos/cosmos-sdk
 COPY . .
 
 # build Cosmos SDK, remove packages
-RUN make tools && \
-    make build-simd && \
+RUN make build-simd && \
     cp ./build/sim* /go/bin
```

```suggestion
    cp ./build/simd /go/bin
```
nit, as there's only one binary for simapp now